.github/workflows/ci.yml

```diff
@@ -43,3 +43,55 @@ jobs:
         run: npm run test:axe
       - name: Unit Tests
         run: npx web-test-runner --config web-test-runner.config.js --group default --playwright --browsers chromium firefox webkit
+  publish:
+    name: Publish static site
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: Brightspace/third-party-actions@actions/checkout
+      - uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+          node-version-file: .nvmrc
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm ci
+      - name: Build
+        run: npm run build
+
+      - name: Build static
+        run: npm run build-static
+
+      - name: Assume role
+        uses: Brightspace/third-party-actions@aws-actions/configure-aws-credentials
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
+          role-to-assume: arn:aws:iam::037018655140:role/brightspace-ui-core.d2l.dev-githubactions-access
+          role-duration-seconds: 3600
+          aws-region: ca-central-1
```

Comment:
I originally had this in `publish-to-s3` directly, but it wasn't allowing me to access `secrets`. Does that sound right? Would be nice to have it in the other action, but might not be possible.

Reply:
Yup this is right - the secrets would need to be passed into the actions as inputs, or as `env` variables